.github/workflows/Test-coverage.yaml

```diff
@@ -48,3 +54,35 @@ jobs:
       - name: Test coverage
         run: covr::codecov(type = "all")
         shell: Rscript {0}
+
+  test-coverage-windows:
```

Comment:
Same as above, just for Windows. We special-case a bunch of Windows stuff, so this made sense to me.

Reply:
You can see that this adds 0.74% to test coverage so I think its worth it.